warehouse/core/src/main/java/datawave/tables/schema/ShardFamilyConstants.java

```diff
@@ -0,0 +1,33 @@
+package datawave.tables.schema;
+
+import org.apache.hadoop.io.Text;
+
+public class ShardFamilyConstants {
+    
+    /**
+     * Document column
+     */
+    public static final String DOCUMENT = "d";
+    public static final Text DOCUMENT_TXT = new Text(DOCUMENT);
+    public static final byte[] DOCUMENT_BYTES = DOCUMENT_TXT.getBytes();
+    // locality group name
+    public static final String FULL_CONTENT_LOCALITY_NAME = "fullcontent";
+    
+    /**
+     * Term frequency column.
+     */
+    public static final String TF = "tf";
+    public static final Text TF_TXT = new Text(TF);
+    public static final byte[] TF_BYTES = TF_TXT.getBytes();
+    // locality group name
+    /* TODO Make a clearer definition of full content indexers */
+    public static final String TERM_FREQUENCY_LOCALITY_NAME = "termfrequency";
+    
+    /**
+     * Fi column
+     */
+    public static final String FI = "fi";
+    public static final Text FI_TXT = new Text(FI);
+    public static final byte[] FI_BYTES = FI_TXT.getBytes();
```

Comment:
`FI_PREFIX` is a common constant with a value of `"fi\0"` or `"fi\u0000"`.

This begs the questions, are these constants purely for the column families or are these a collection of table constants?

Reply:
I dig the PREFIX.  Will add.
I was going to start with family related constants and see what crawls out of the woodwork for another MR for other things.